src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -236,7 +235,7 @@ public function setLoggers(array $loggers): array
 
         if ($flush) {
             foreach ($this->bootstrappingLogger->cleanLogs() as $log) {
-                $type = $log[2]['exception'] instanceof \ErrorException ? $log[2]['exception']->getSeverity() : E_ERROR;
+                $type = $log[2]['exception'] instanceof \ErrorException ? $log[2]['exception']->getSeverity() : ThrowableUtils::getSeverity($log[2]['exception']);
```

Comment:
you could handle `\ErrorException` inside `ThrowableUtils::getSeverity`. seems cleaner to me.